### PR TITLE
Added CodeCov again

### PR DESCRIPTION
**Pull request by Alexander-Sol**
Test post, trying to get CodeCov checks to show automatically

### CODE REVIEW

**Comment by acesnik** [comment]
Here's how I did this for `topdownsdk` recently, in case that's helpful: https://github.com/topdownproteomics/sdk/pull/94/files

**Comment by Alexander-Sol** [comment]
> Here's how I did this for `topdownsdk` recently, in case that's helpful: https://github.com/topdownproteomics/sdk/pull/94/files

I think we're communicating with CodeCov, but the issue is that checks aren't being generated automatically on github. I'm not sure if it's an issue with github actions or one of the external workflows. But I didn't see any checks on the repo you linked either.

**Comment by acesnik** [comment]
Looks like comments are off by default in the codecov settings for smith-chem-wisc
image

**Comment by acesnik** [comment]
image

**Comment by acesnik** [comment]
Tried that, but to no avail.

File: .github/workflows/dotnet.yml [hunk 1]
```diff
@@ -25,15 +27,17 @@ jobs:
...
     - name: Test
-      run: cd mzLib && dotnet test --no-build --verbosity normal --collect:"XPlat Code Coverage" /p:CoverletOutputFormat=cobertura ./Test/Test.csproj
+      run: cd mzLib/Test && dotnet test --no-build --verbosity normal 
+        --collect:"XPlat Code Coverage" /p:CoverletOutputFormat=cobertura
     - name: TestFlashLFQ
-      run: cd mzLib && dotnet test --no-build --verbosity normal --collect:"XPlat Code Coverage" /p:CoverletOutputFormat=cobertura ./TestFlashLFQ/TestFlashLFQ.csproj
-    - name: Codecov
+      run: cd mzLib/TestFlashLFQ && dotnet test --no-build --verbosity normal --collect:"XPlat Code Coverage" /p:CoverletOutputFormat=cobertura
+    - name: Upload to Codecov
       uses: codecov/codecov-action@v2
```
Comment (acesnik) [inline .github/workflows/dotnet.yml:39]: could try setting this to v3

File: .github/workflows/dotnet.yml [hunk 2]
```diff
@@ -25,15 +27,17 @@ jobs:
...
-      run: cd mzLib && dotnet add TestFlashLFQ/TestFlashLFQ.csproj package coverlet.collector
+      run: cd mzLib/TestFlashLFQ && dotnet add package coverlet.collector
     - name: Test
-      run: cd mzLib && dotnet test --no-build --verbosity normal --collect:"XPlat Code Coverage" /p:CoverletOutputFormat=cobertura ./Test/Test.csproj
+      run: cd mzLib/Test && dotnet test --no-build --verbosity normal 
+        --collect:"XPlat Code Coverage" /p:CoverletOutputFormat=cobertura
     - name: TestFlashLFQ
-      run: cd mzLib && dotnet test --no-build --verbosity normal --collect:"XPlat Code Coverage" /p:CoverletOutputFormat=cobertura ./TestFlashLFQ/TestFlashLFQ.csproj
-    - name: Codecov
+      run: cd mzLib/TestFlashLFQ && dotnet test --no-build --verbosity normal --collect:"XPlat Code Coverage" /p:CoverletOutputFormat=cobertura
```
Comment (acesnik) [inline .github/workflows/dotnet.yml:37]: The TestFlashLFQ coverage dropped a bunch, so I'm not sure this is working https://app.codecov.io/gh/smith-chem-wisc/mzLib/pull/669

**Comment by acesnik** [comment]
Oh, the topdownsdk one did work for one PR after mine with the codecov stuff, https://github.com/topdownproteomics/sdk/pull/95, but then codecov must have changed something.

**Comment by acesnik** [comment]
What do you think about keeping the `.github/codecov.yaml` file with your new settings here, as well? The global settings on Codecov are a bit opaque and not version controlled?

File: .github/workflows/dotnet.yml [hunk 3]
```diff
@@ -25,15 +27,18 @@ jobs:
     - name: Build (TestFlashLFQ)
       run: cd mzLib && dotnet build --no-restore ./TestFlashLFQ/TestFlashLFQ.csproj
     - name: Add coverlet collector (Test)
```
Comment (acesnik) [inline .github/workflows/dotnet.yml:29]: Can these be removed because you added the coverlet PackageReference to these csproj files?

Comment (Alexander-Sol) [reply .github/workflows/dotnet.yml:29]: I think you're right. However, the changes I made in this pull request were unnecessary and broke test coverage in FlashLFQ.

**Comment by Alexander-Sol** [comment]
CodeCov was fixed by enabling the CodeCov github app here: https://github.com/organizations/smith-chem-wisc/settings/installations

The issue wasn't in the .yaml, but instead with the app permission settings.

**Comment by acesnik** [comment]
What setting did you change? I'm trying to fix this for another repo

**Pull request closed without merging**